integration-tests/main/src/test/java/io/quarkus/it/main/MockitoTest.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.it.main;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class MockitoTest {
+
+    @Test
+    public void testMockHttpServletRequest() {
+        HttpServletRequest request = mock(HttpServletRequest.class);
```

Comment:
How about using AssertJ like this? 

```
assertThatCode(() -> mock(HttpServletRequest.class)).doesNotThrowAnyException(); 
```

Reply:
Thought about it, but adding AssertJ just for this didn't seem to warrant it. 
But I guess I should do it since AssertJ would provide a much better error message if mockito does start failing...

Reply:
Done